teamsalerts/main.tf

```diff
@@ -1,5 +1,6 @@
 module "sns_to_teams" {
-  source  = "github.com/massgov/mds-terraform-common//lambda?ref=1.0.26"
+  # TODO: change branch to tag before deploying
+  source  = "github.com/massgov/mds-terraform-common//lambda?ref=DP-27416-terraform-upgrade-0.13"
```

Comment:
ditto this: https://github.com/massgov/mds-terraform-common/pull/95/files#r1137388680